criticalmaps/criticalmaps-ios
# Add UITest Infrastructure

Fixes #84. 

This includes the following changes: 
- Add new Target for UITests
- Add UITest which navigates through all screens and tests the navigation, if all screens are reachable
- Add fastlane action to run `bundle exec fastlane screenshots` to generate screenshots from every screen in all available languages and different devices
- Check in the configurated schemes to run UI Tests separately from Unit Tests 
   - CriticalMaps
   - CriticalMapsTests
   - CriticalMapsUITests
- Update Fastlane

## Checklist

### Before merging the PR

- [x] If applicable, does the PR contain enough unit / UI tests?
- [x] If applicable, did you add a before / after screenshot of the feature?
- [x] Did you update the [CHANGELOG.md](https://github.com/criticalmaps/criticalmaps-ios/blob/master/CHANGELOG.md)?
